nodes.py

```diff
@@ -11,11 +11,13 @@
 import numpy as np
 
 sys.path.insert(0, os.path.join(sys.path[0], "comfy"))
+sys.dont_write_bytecode = True #No __pycache__ plz
```

Comment:
Can't you just ignore the __pycache__ files in your code that loads the folders?

Reply:
You're right